docs/lit/examples/1_Examples.jl

```diff
@@ -52,6 +52,25 @@ plot(y, ylabel=P"mass in kilograms")
 
 plot(y, ylabel="")
 
+# ### Surround
+
+# If you prefer some other formatting over the round parentheses, you can
+# supply a keyword `surround`, which can be a number of different things:
+# `:round`,`:square`,`:curly` and `:slash` do what you might expect.
+# `:slashround` and the likes will combine them. `:verbose` will insert an " in
+# units of " between the label and the unit. `nothing` will remove the
+# parentheses.  For more customizatbility, a character, a string or a `Tuple`
```

Comment:
I think it would be good to cut the paragraph after "parentheses." here and have Julia example blocks with these symbols.

Reply:
Maybe good place for something like I suggested in the conversation, with many subplots each corresponding to a different symbol, but maybe with `"surround=symbol"` as the title, like
```julia
symbols = [:round, :square, :curly, :angle, :slash, :slashround, :slashsquare, :slashcurly, :slashangle, :verbose, nothing]
plot([plot(y, title="surround=$s", surround=s) for s in symbols]...)
```